olive/systems/docker/docker_system.py

```diff
@@ -141,9 +141,11 @@ def _run_container(self, tempdir, model: OliveModel, metrics: List[Metric], cont
         try:
             logger.debug(f"Running container with eval command: {eval_command}")
             logger.debug(f"The volumes list is {volumes_list}")
-            self.docker_client.containers.run(
-                image=self.image, command=eval_command, volumes=volumes_list, **run_command
+            container = self.docker_client.containers.run(
+                image=self.image, command=eval_command, volumes=volumes_list, detach=True, **run_command
             )
+            for line in container.logs(stream=True):
+                print(line.strip().decode())
```

Comment:
logger.info will swallow the log outputs in the docker container. Use print here instead.